Modules/install-jaspBase.R.in

```diff
@@ -4,12 +4,14 @@ Sys.setenv(GITHUB_PAT="@GITHUB_PAT@")
 Sys.setenv(RENV_PATHS_ROOT="@MODULES_RENV_ROOT_PATH@")
 Sys.setenv(RENV_PATHS_CACHE="@MODULES_RENV_CACHE_PATH@")
 Sys.setenv(JASPENGINE_LOCATION="@JASP_ENGINE_PATH@/JASPEngine")
+options(renv.config.install.verbose = @RENV_INSTALL_VERBOSE@)
```

Comment:
Dont we always want it verbose during this stage?

Reply:
At this stage, yes, I'll change that.